iwyu_tool_test.py

```diff
@@ -363,6 +363,34 @@ def test_extra_args_with_sep(self):
         self.assertEqual(['arg1', '--', 'another_arg1'],
                          self.main.call_args['extra_args'])
 
+    def test_jobs_normal(self):
+        """ Nonzero values for -j are forwarded as they are. """
+        argv = ['iwyu_tool.py', '-p', '.', '-j', '4']
+        iwyu_tool._bootstrap(argv)
+        self.assertEqual(4, self.main.call_args['jobs'])
+
+
+    def test_jobs_zero(self):
+        """ -j=0 will not hang the program. """
+        argv = ['iwyu_tool.py', '-p', '.', '-j', '0']
+        iwyu_tool._bootstrap(argv)
+        self.assertNotEqual(0, self.main.call_args['jobs'])
+        self.assertNotEqual(None, self.main.call_args['jobs'])
```

Comment:
Prefer `assertIsNotNone` for None-checking: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIsNotNone